cli/lib/kontena/cli/stacks/list_command.rb

```diff
@@ -35,9 +48,13 @@ def fields
     end
 
     def execute
-      print_table(stacks) do |row|
+    stacks = get_stacks
+    build_depths(stacks)
+    stacks = stacks.sort_by { |s| s['name'] }
```

Comment:
Does it make any difference?

The tree drawing thing confuses myself too and it works by strange magic or pure luck.

Reply:
I reckon it works, assuming that the parent stack always comes before any child stacks.

Hence the `sort`, because the child stack names always have the parent stack name as their prefix, so they will sort lexically after the parent stack.

Reply:
It won't matter if the depths are built before or after sorting

Reply:
At least I don't see how

Reply:
It's the same for single-level dependencies, but broken for higher-level dependencies unless sorted? 

With sorted `top1`, `child2`, `subchild3` => depths 1, 2, 3
With reversed `subchild3`, `child2`, `top1` => depths 2, 2, 1?

Reply:
Seems so. Fixed.